app/src/main/java/com/example/alexandraboukhvalova/a436msclinicalmonitoring/instruction.java

```diff
@@ -0,0 +1,29 @@
+package com.example.alexandraboukhvalova.a436msclinicalmonitoring;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+public class instruction extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        final Context context = this;
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_instruction);
+        final TextView tv=(TextView)findViewById(R.id.textView1);
+
+
+        tv.postDelayed(new Runnable() {
+            public void run() {
+                Intent intent = new Intent(context, TappingActivity.class);
+                startActivity(intent);
+
+            }
+        }, 7000);
```

Comment:
I see that the user waits seven seconds at the instruction activity before proceeding to the activity where the test resides. I found an issue where if the user hits "back", the intent still follows through with changing the activity. I believe this is an issue with maintaining the back stack. For this reason, maybe giving the instructions their own activity is not the direction we want to head.